admin/app/controllers/admin/InteractiveLibrarianController.scala

```diff
@@ -18,6 +18,28 @@ class InteractiveLibrarianController(
     with GuLogging
     with ImplicitControllerExecutionContext {
 
+  def pressForm(): Action[AnyContent] =
+    Action { implicit request =>
+      // This view is displayed on the admin tool
```

Comment:
Minor, but this comment feels redundant given the controller already lives within the Admin app.

Reply:
Ok, will remove, thanks!